src/app/features/tech-record/create-batch-trl/components/batch-trl-results/batch-trl-results.component.ts

```diff
@@ -0,0 +1,51 @@
+import { Component, OnInit } from '@angular/core';
+import { Router, ActivatedRoute } from '@angular/router';
+import { StatusCodes } from '@models/vehicle-tech-record.model';
+import { TechnicalRecordService } from '@services/technical-record/technical-record.service';
+import { take } from 'rxjs';
+
+@Component({
+  selector: 'app-batch-trl-results',
+  templateUrl: './batch-trl-results.component.html',
+  styleUrls: ['./batch-trl-results.component.scss']
+})
+export class BatchTrlResultsComponent implements OnInit {
+  constructor(private technicalRecordService: TechnicalRecordService, private router: Router, private route: ActivatedRoute) {}
+
+  ngOnInit() {
```

Comment:
These, again, do not depend on inputs, so they can be moved into the constructor.